com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDAdditionalCameraData.cs

```diff
@@ -13,7 +13,7 @@ public class HDPhysicalCamera
         /// <summary>
         /// The minimum allowed aperture.
         /// </summary>
-        public const float kMinAperture = 1f;
+        public const float kMinAperture = 0.7f;
```

Comment:
I am almost certain it will work fine, but worth checking how changing this affects existing projects